sdk/src/main/scala/io/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -264,7 +264,7 @@ class AccountForgeMessageBuilder(
               )
               require(ommers.isEmpty, "No Ommers allowed for the last block of the withdrawal epoch.")
 
-              val withdrawalEpochNumber: Int = dummyView.getWithdrawalEpochInfo.epoch
```

Comment:
Why wasn't taking the withdrawal epoch from the view correct?

Reply:
In case first MC block of the next epoch haven't been added to Sidechain, view consider that it's previous epoch now. When we include all MC blocks from the epoch it causes that fee will be calculated for previous epoch,